hera_qm/utils.py

```diff
@@ -276,6 +276,36 @@ def get_metrics_ArgumentParser(method_name):
                         'visibilities formed with these antennas will be set to True.')
         ap.add_argument("--clobber", default=False, action="store_true",
                         help='overwrites existing files (default False)')
+    elif method_name == 'xrfi_run_data_only':
+        ap.prog = 'xrfi_run_data_only.py'
+        ap.add_argument('--data_file', default=None, type=str, help='Raw visibility '
+                        'data file to flag on.')
+        ap.add_argument('--xrfi_path', default='', type=str,
+                        help='Path to save flag files to. Default is same directory as input file.')
+        ap.add_argument('--kt_size', default=8, type=int,
+                        help='Size of kernel in time dimension for detrend in xrfi '
+                        'algorithm. Default is 8.')
+        ap.add_argument('--kf_size', default=8, type=int,
+                        help='Size of kernel in frequency dimension for detrend in '
+                        'xrfi algorithm. Default is 8.')
+        ap.add_argument('--sig_init', default=6.0, type=float,
+                        help='Starting number of sigmas to flag on. Default is 6.0.')
+        ap.add_argument('--sig_adj', default=2.0, type=float,
+                        help='Number of sigmas to flag on for data adjacent to a flag. Default is 2.0.')
+        ap.add_argument('--ex_ants', default=None, type=str,
+                        help='Comma-separated list of antennas to exclude. Flags of visibilities '
```

Comment:
This only works if there are no spaces right? Should probably clarify that.

Reply:
Do you mean that `ex_ants` won't work if there are not spaces in the comma-separated list of antennas? I believe the other ex_ants parsers also neglect to mention this. I can update all of them if you want .

Reply:
Nevermind, it works regardless of spaces as long as there are also commas.